.github/workflows/nix.yaml

```
@@ -22,38 +22,25 @@ jobs:
     outputs:
-      should-build: ${{ steps.evaluate.outputs.should_build }}
-      should-push-docker: ${{ steps.evaluate.outputs.should_push_docker }}
+      should-proceed: ${{ steps.evaluate.outputs.should_proceed }}
       has-backend: ${{ steps.check-labels.outputs.has_backend }}
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      
       - name: Apply labels
         if: github.event_name == 'pull_request' && github.event.action != 'closed'
         uses: actions/labeler@v4
```

actionlint (1.7.4)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)